### DISCUSSION TITLE
Adding an extension, for an audio format that is handled by ffmpeg

### DISCUSSION THREAD

**Discussion by ghost** [Tichau/FileConverter#138] [Q&A] [answered]
Hello @Tichau  

First thanks for this nice and efficient converter (and Happy New Year)

The 64bit version of wave format is often saved with extension **w64,** and not wav.
Is it possible currently to make FileConverter accept the **w64** extension (for convertion to other formats, like FLAC, m4a etc) ?

**Comment by ghost** [comment, marked as answer]
I found a solution to fix that, but it implies to edit a value in registry:
Browse registry to

[HKEY_CURRENT_USER\SOFTWARE\FileConverter]

Then add **w64** in the list of "CompatibleInputExtensions".

The extension **w64** appears now in the settings!

**Reply by tryrebooting** [reply]
Awesome find. Thanks!
I was able to add **opus** to be able to convert to various audio formats.
